meta-cli/src/typegraph/loader/mod.rs

```diff
@@ -87,7 +97,10 @@ impl<'a> Loader<'a> {
 
     async fn load_command(&self, mut command: Command, path: &Path) -> LoaderResult {
         let path: Arc<Path> = path.into();
+        let path: Arc<Path> = path.clone().into();
```

Comment:
Huh? Why the `clone`?

Reply:
My bad, I did some tests then forgot removing this line.